### PR TITLE
Use babel-loader to enable code-splitting of bundles

### PR DESCRIPTION
**Pull request by timbrown5**
Added babel-loader and switch to it for jsx? and TSX? files.

Bundles before:
Screenshot 2023-02-23 at 14 41 14

Bundles after:
Screenshot 2023-02-23 at 14 43 33

Command:
`./node_modules/.bin/webpack --profile --json > stats.json ; ./node_modules/.bin/webpack-bundle-analyzer ./stats.json`

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,14 +27,17 @@
     "@mui/material": "5.10.10",
     "@react-spring/web": "9.6.0",
     "@reacticons/ionicons": "6.0.4",
+    "bable-loader": "^0.0.1-security",
```
Comment (timja) [inline package.json:30]: typo squatted package? https://www.npmjs.com/package/bable-loader

**Review by timja — CHANGES_REQUESTED** [review]

**Comment by timja** [comment]
What are the numbered ones at the start?

Pipeline console view looks a lot smaller

**Comment by timbrown5** [comment]
Before:
Screenshot 2023-02-23 at 15 27 27

After:
Screenshot 2023-02-23 at 15 29 24

**Comment by timbrown5** [comment]
Not sure why iconList.json takes up so much space in console view?

**Comment by timbrown5** [comment]
Here is a before and after video of the lazy loading (with network throttling enabled):
Before:
https://user-images.githubusercontent.com/4447764/221029987-1f8cfd03-73d4-4908-9c4f-a99b22cebcf1.mov

After:
https://user-images.githubusercontent.com/4447764/221030009-9bcc951a-6f1b-4514-8830-92e4103a7871.mov

**Comment by timja** [comment]
Wow nice, anything remaining on this?

**Comment by timbrown5** [comment]
I'm happy it solves the large bundle problem, but not sure if there's a
better/preferred way to do it?
Also not sure if it breaks anything (I did a cursory look at the console
and graph gives and they render as expected).

**Comment by timja** [comment]
I haven't found any issues, couldn't really notice any difference.

When I had slow 3g throttle on I was getting faster on the original and 1mb bundle instead of 600 or so kb on the network tab.

but happy to try this out anyway.

**Review by timja — APPROVED** [review]

**Comment by timbrown5** [comment]
I've found some additional savings - mostly removing ionicons (which was causing the large iconList.json file). Size is down from 2.MB to 1.28MB.

Screenshot 2023-03-09 at 22 24 55

I believe I fixed the scaling issues with the SvgStatus class (which allowed me to remove icoicons). It also adds the correct colors to the DataTreeView:
Before:
Screenshot 2023-03-09 at 22 28 39

After: 
Screenshot 2023-03-09 at 22 27 57

**Comment by timja** [comment]
nice!

File: src/main/frontend/pipeline-console-view/pipeline-console/main/PipelineConsoleModel.tsx [hunk 2]
```diff
@@ -8,6 +8,9 @@ export {
 } from "../../../pipeline-graph-view/pipeline-graph/main/PipelineGraphModel";
 
 export { getGroupForResult } from "../../../pipeline-graph-view/pipeline-graph/main/support/StatusIcons";
+
+//export { default as StepStatus } from "../../../step-status/StepStatus";
```
[the remark this answers is not in the document]
Comment (timja) [reply src/main/frontend/pipeline-console-view/pipeline-console/main/PipelineConsoleModel.tsx:12]:
```suggestion
```

**Pull request merged**